### DISCUSSION TITLE
'filetype' yaml rejected by internal_filetypes[yaml]

### DISCUSSION THREAD

**Discussion by shnee** [community/community#118856] [Copilot Conversations]
### Select Topic Area

Question

### Body

When using the vim copilot plugin I get this error for YAML files. The plugin works great on other file types.

Putting this in my vimrc does not change the behavior.
```vim
let g:copilot_filetypes = {'yaml': v:true}
```

Any ideas?

**Comment by tpope** [comment]
Are you on the latest version? The internal defaults changed to allow YAML back in February, so that config shouldn't even be necessary.

**Comment by shnee** [comment]
It appears that I was not even running the official plugin. I was running this instead zbirenbaum/copilot.lua. Closing.
